core/config.go

```diff
@@ -0,0 +1,19 @@
+// Copyright © 2022 Obol Labs Inc.
+//
+// This program is free software: you can redistribute it and/or modify it
+// under the terms of the GNU General Public License as published by the Free
+// Software Foundation, either version 3 of the License, or (at your option)
+// any later version.
+//
+// This program is distributed in the hope that it will be useful, but WITHOUT
+// ANY WARRANTY; without even the implied warranty of  MERCHANTABILITY or
+// FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+// more details.
+//
+// You should have received a copy of the GNU General Public License along with
+// this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package core
+
+// BuilderEnabled determines whether the builderAPI is enabled for the provided slot.
+type BuilderEnabled func(slot int64) bool
```

Comment:
is builderAPI enabled per slot? I didn't understand the relation between these two

Reply:
yes, prioritise enables/disables config based on slot, so from slot x to slot y it is enabled, otherwise disabled.